Dockerfile.dapper

```diff
@@ -4,7 +4,7 @@ ARG DAPPER_HOST_ARCH
 ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH}
 ENV CATTLE_HELM_VERSION v2.16.8-rancher1
 ENV CATTLE_MACHINE_VERSION v0.15.0-rancher73
-ENV CATTLE_K3S_VERSION v1.22.3+k3s1
+ENV CATTLE_K3S_VERSION v1.22.4+k3s1
```

Comment:
Could you help me understand why the k3s version was bumped from 1.22.3, which was where I left off with embedded etcd, to 1.22.5 and back to 1.22.4. What changed?

Reply:
I see it as a bump from 1.22.3 to 1.22.4 to 1.22.5. Not seeing where it was set to 1.22.4 again...

Reply:
@annablender I think you're reviewing one of the intermediate commits. The latest has it bumped to 1.22.5.